protoc_plugin/lib/file_generator.dart

```diff
@@ -570,12 +570,20 @@ class FileGenerator extends ProtobufContainer {
 
   /// Writes an import of a .dart file corresponding to a .proto file.
   /// (Possibly the same .proto file.)
+  ///
+  /// When generating the main file (if [forMainFile] is true), all imports
```

Comment:
What do you want to omit the import prefix?
Can the logic be simplified by never not using a prefix?

Reply:
I am also a bit confused. I think we might as well always prefix...

Reply:
Done, might as well always prefix as you both suggested.